#!/bin/sh /usr/share/dpatch/dpatch-run
## 80-kolab-syslog-facility.dpatch by Marc Brockschmidt <marc.brockschmidt@credativ.de>
##
## DP: Make the used syslog facility configureable

@DPATCH@
diff -Nwru kolabd-1.9.4dfsg/kolabconf kolabd-1.9.4dfsg.patched/kolabconf
--- kolabd-1.9.4dfsg/kolabconf	2005-04-28 02:16:59.000000000 +0200
+++ kolabd-1.9.4dfsg.patched/kolabconf.in	2006-05-26 15:32:13.000000000 +0200
@@ -37,7 +37,7 @@
 use Kolab::LDAP;
 use vars qw($opt_d $opt_n $opt_h);
 
-openlog('kolabconf', 'cons, pid', 'user');
+openlog('kolabconf', 'cons, pid', ($Kolab::config{'syslog_facility'} || 'user'));
 
 getopts('dnh');
 if ($opt_h) {
diff -Nwru kolabd-1.9.4dfsg/kolab.conf kolabd-1.9.4dfsg.patched/kolab.conf
--- kolabd-1.9.4dfsg/kolab.conf	2004-11-23 21:26:47.000000000 +0100
+++ kolabd-1.9.4dfsg.patched/kolab.conf	2006-05-26 15:21:52.000000000 +0200
@@ -10,3 +10,4 @@
 ldap_master_uri : @@@ldap_master_uri@@@
 php_dn : cn=nobody,@@@kolab_basedn@@@
 php_pw : @@@nobody_pw@@@
+syslog_facility : local5
diff -Nwru kolabd-1.9.4dfsg/kolabd kolabd-1.9.4dfsg.patched/kolabd
--- kolabd-1.9.4dfsg/kolabd	2004-12-22 14:50:03.000000000 +0100
+++ kolabd-1.9.4dfsg.patched/kolabd.in	2006-05-26 15:31:56.000000000 +0200
@@ -49,7 +49,7 @@
 use Kolab::LDAP::Backend;
 use vars qw(%pids);
 
-openlog('kolabd', 'cons, pid', 'user');
+openlog('kolabd', 'cons, pid', ($Kolab::config{'syslog_facility'} || 'user'));
 
 my $prefix = $Kolab::config{'prefix'};
 my $pidfile = IO::File->new("$prefix/var/kolab/kolab.pid", 'w+')
